html/readingTherapy.html

```diff
@@ -53,7 +53,47 @@ <h3 id="subtitle">Read Articles, Motivational Quotes and listen to summaries of
       <section id="articles">
       <h1 style="font-size:2.3rem">Articles</h1>
       <h2>Inspirational Stories</h2>
-    <div class="collection">
+      <div class="workcard">
+        <div class="collage-row">
+          <div class="collage-card">
+            <img src="../images/bestwaytorefresh.jpg"></img>
+            <div class="project-hov">
+              <h2 class="text">The Best Way to Refresh Mind, Body & Soul</h2>
+              
+              <button class="butn-red"><a href="https://ahigherthought.com/the-best-way-to-refresh-mind-body-soul/" target="_blank">Read More</a></button>
+      
+            </div>
+          </div>
+          <div class="collage-card">
+            <img src="../images/31inspiration.jpg"></img>
+            <div class="project-hov">
+              <h2 class="text">31 Best Inspirational & Motivational Stories</h2>
+              
+              <button class="butn-red"><a href="https://www.developgoodhabits.com/inspirational-stories/" target="_blank">Read More</a></button>
```

Comment:
Can we change the class name `butn-red` to `btn-red`?

Most of the developer use btn, and easy to read.